Source/WebCore/Modules/mediasource/SourceBuffer.cpp

```diff
@@ -1704,15 +1706,18 @@ void SourceBuffer::sourceBufferPrivateDidReceiveSample(SourceBufferPrivate*, Med
         if (m_shouldGenerateTimestamps)
             m_timestampOffset = frameEndTimestamp;
 
+        if (trackBuffer.fudgeRoom.isInvalid() || trackBuffer.fudgeRoom < frameDuration)
+            trackBuffer.fudgeRoom = frameDuration;
```

Comment:
I pushed a fix for this issue in the yttv-2016-45-fixed branch some months ago, but I don't know if it still applies to current master: https://github.com/WebPlatformForEmbedded/WPEWebKit/commit/d049232274ac042613fbb334002300a4f1b809a1

However, it never became integrated in master because we preferred not to touch WebCore code in order to keep the differences with upstream WebKit as small as possible.

When I coded it, I found this comment in the code suggesting that the fudgeFactor value was related to video frame duration. I thought that assumption wouldn't fit well for audio frames.